specifications/16-case-insensitive-in-operators.json

```diff
@@ -0,0 +1,200 @@
+{
+  "name": "16-case-insensitive-in-operators",
+  "state": {
+    "version": 2,
+    "features": [
+      {
+        "name": "F9.globalSegmentAndConstraint_IN",
+        "description": "With global segment and constraint both on",
+        "enabled": true,
+        "strategies": [
+          {
+            "name": "default",
+            "parameters": {},
+            "constraints": [
+              {
+                "contextName": "appName",
+                "operator": "IN",
+                "caseInsensitive": true,
+                "inverted": false,
+                "values": [
+                  "app"
+                ]
+              }
+            ],
+            "segments": [1]
+          }
+        ]
+      },
+      {
+        "name": "F9.globalSegmentAndConstraint_IN_inverted",
+        "description": "With global segment and constraint both on",
```

Comment:
The description here seems to be the same for each toggle, can we give these a more unique name describing what they do? They used in some of the SDK test scaffolds to report which test failed so it's useful if they're unique and descriptive

Reply:
Thanks makes sense